tests/test_cooling_effect.py

```diff
@@ -1,32 +1,30 @@
-import numpy as np
 from pythermalcomfort.models import cooling_effect
 
-def test_cooling_effect(get_cooling_effect_url, retrieve_data):
+
+def test_cooling_effect(get_cooling_effect_url, retrieve_data, is_equal):
 
     reference_table = retrieve_data(get_cooling_effect_url)
-    
+    tolerance = reference_table["tolerance"]
 
     for entry in reference_table["data"]:
         inputs = entry["inputs"]
         expected_output = entry["outputs"]["cooling_effect"]
-        
-    
+
         result = cooling_effect(
             tdb=inputs["tdb"],
             tr=inputs["tr"],
             rh=inputs["rh"],
             vr=inputs["vr"],
             met=inputs["met"],
             clo=inputs["clo"],
-            units=inputs.get("units", "SI") 
+            units=inputs.get("units", "SI"),
         )
-        
+
         # To determine whether the result is as expected, use np.allclose for arrays and np.isclose for single
         try:
-            if isinstance(expected_output, list):
-                assert np.allclose(result, expected_output, atol=0.1), f"Expected {expected_output}, got {result}"
-            else:
-                assert np.isclose(result, expected_output, atol=0.1), f"Expected {expected_output}, got {result}"
+            is_equal(result, expected_output, tolerance.get("cooling_effect"))
```

Comment:
this should be fixed since if the key is not there then this may throw an issue.